app/views/transfers/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="new-transaction-container">
+  <div class="header">
+    <%= link_to '←', category_path(@category), class: 'back-link' %>
+    <h1 class="page-title">New Transaction</h1>
+  </div>
+
+  <div class="transaction-form">
+    <%= form_for :transfer, url: category_transfers_path(@category), method: :post, html: { class: 'transaction-form' } do |f| %>
+      <div class="field">
+        <%= f.text_field :name, id: 'name', autofocus: true, autocomplete: "name", placeholder: 'Name', required: true %>
+      </div>
+
+      <div class="field">
+        <%= f.text_field :amount, id: 'amount', autofocus: true, autocomplete: "amount", placeholder: 'Amount', required: true %>
+      </div>
```

Comment:
- [optional] Kindly take note that users are expected to choose out of the categories created. I will highly suggest that you implement it in order to solidify your knowledge. 🙏 

<img width="409" alt="Untitled" src="https://github.com/ahmidou09/budget_manager/assets/103386183/8c98b99a-f77e-4235-a43d-d567e232846d">